CONTRIBUTING.md

```diff
@@ -146,19 +150,17 @@ pytest tests/test_290_system_consumers.py -v
 pytest tests/test_300-system_roles.py -v
 pytest tests/test_310-system_permissions.py -v
 pytest tests/test_320-settings_banner.py -v
-
 pytest tests/test_990-delete_all_resources.py -v
-~~~
+```
 
-Or you can simply run `pytest -v` to test everything all at once. The above commands however allow you to halt
-testing to fix issues that might arise.
+Or you can simply run `pytest -v` to test everything all at once. The above commands however allow you to halt testing
+to fix issues that might arise.
 
 ## Secrets Manager Testing
 
-Since the admin API actions for Secrets Manager are not yet built into this Python API Wrapper then
-some manual setup has to occur in the tenant used for testing. Specifically these tests will need at
-least 1 secret available to the calling user/service identity and one of those secrets has to be named
-"/Test".
+Since the admin API actions for Secrets Manager are not yet built into this Python API Wrapper then some manual setup
```

Comment:
this is now in the sdk. we can remove these notes i think. the test suite tests the end to end secrets manager processes.